specification.md

```diff
@@ -1075,9 +1077,167 @@ Note that the referenced function definition type in this case must be `rpc`.
 
 For more information about functions, reference the [Functions definitions](#Function-Definition) section.
 
+### Using Functions For GraphQL Service Invocations
+
+If you want to use GraphQL services, you can also invoke them using a similar syntax to the above methods with the caveat that your `functionRef`'s `arguments` property must have the following properties:
+
+| Parameter | Description | Type | Required |
+| --- | --- | --- | --- |
+| inputs | object whose keys are the field's arguments name, and its value that argument value | object | no |
```

Comment:
Should we change maybe to:

"Input object containing field argument names and their values"